clean up use of word: easy (#26352)
kubernetes/website: content/en/docs/tutorials/stateless-application/guestbook.md

## Scale the Web Frontend

> Scaling up or down is effortless because your servers are defined as a Service that uses a Deployment controller.

Reviewer: ```suggestion
You can scale up or down as needed because your servers are defined as a Service that uses a Deployment controller.
```
